README.rst

```diff
@@ -39,13 +39,20 @@ We recommend you use a virtual environment for running `bigchaindb-driver`, to i
 .. code-block:: text
 
     pip install -U bigchaindb-driver
-    
+
 If this fails, ensure you have python 3 installed, if you have both versions of python installed, to install and update using `pip`:
 
 .. code-block:: text
 
     pip3 install -U bigchaindb-driver
 
+Get Started with BigchainDB Server
+------------------------------------
+* `Set Up & Run a Dev/Test Node`_
+* `Run BigchainDB Server with Docker`_
+* `Run BigchainDB Server with Vagrant`_
+* `Run BigchainDB Server with Ansible`_
+
```

Comment:
These links might be defunct or not-working now. I have it in my notes to revisit the same set of links in the README.md file in bigchaindb/bigchaindb

Reply:
oh, do you have them somewhere?

Reply:
In general I would prefer to have just two links. One for dev/test node and one that points to different ways to run bdb. I checked and we don't really have it. Do you know when you will revisit them? Maybe I can wait.

Reply:
It's not clear to me today what all the "standard" ways to run a BigchainDB node will be. It seems the "Docker Compose way" might be one of them. Our Kubernetes-based way is for running a production node. Beyond that, who knows? A link for the Docker Compose way is:

https://docs.bigchaindb.com/projects/contributing/en/latest/dev-setup-coding-and-contribution-process/run-node-with-docker-compose.html

There's also a page about running a node as processes:

https://docs.bigchaindb.com/projects/contributing/en/latest/dev-setup-coding-and-contribution-process/run-node-as-processes.html

I'm reluctant to maintain a long list of "ways" because that becomes a maintenance nightmare.

Reply:
I agree. It would be good to have one link, that lists/links to possibilities